db/sql/table_partial.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS partial;
```

Comment:
Remove the doc_type column in the template table.

With this change, every row in the template table will have a value of "enrollment" for doc_type, as that's the only supported document type.  I understand we may have to support multiple document types in the future, however that's speculative programming.

Reply:
I see what you're saying, but I disagree with the statement "that's the only supported document type". All document types are supported by DTS, we just happen to be using only one. Having a doc type here prevents us from having to load in every single partial for every template doc_type. I wouldn't call it speculative programming here, it was part of the initial design, we just implemented it poorly with the whole "enrollment/partial" setup.